docs/en/changelog.md

```diff
@@ -1,5 +1,122 @@
 ## Changelog
 
+### v2.25.0 (31/5/2022)
+
+#### Highlights
+
+- Support [ConvNeXt](https://arxiv.org/abs/2201.03545) (#7281)
```

Comment:
highlights do not need to post PR ID, and they should be consistent with those in README.md.